docs/en/academic/citations.md

```diff
@@ -4,15 +4,15 @@ Citing in Zettlr is done using `citeproc-js`, a library that works like Pandoc's
 
 ## Enabling Citations in Zettlr
 
-There are two engines related to citations: previews (citations can be previewed as images or links) and the actual process of generating citations (which happens only on export). Both of these engines are triggered by selecting a citation library that contains references. Without such a library, Zettlr will preview citations (so that you can see what will trigger Pandoc's citeproc), but won't replace them with formatted citations. Also, if you do not specify such a library, Zettlr will _not_ run Pandoc with its citeproc-engine, and therefore will not parse the citations.
+The citation engine that powers Zettlr is triggered when you point Zettlr to a citation library containing references. If a citation library is not specified, Zettlr will _not_ run Pandoc with its citeproc-engine, and therefore will not parse the citations.
 
-So the first step is to create citation library. Zotero is the recommended application for managing your library, so this tutorial assume's you are using Zotero. If you use another program, you will need to figure out how to export to the CSL JSON format.
+Zotero is the recommended application for managing your library and generating your citation library. If you use another program, you will need to figure out how to export to the CSL JSON format.
```

Comment:
It would be great to add JabRef here, I missed that one initially. JabRef can export BibTex files and that also works fine with Zettlr (so CSL JSON is recommended, but not required, BibTex also works)